lib/MediaInfo.js

```diff
@@ -508,14 +508,25 @@ class MediaInfo {
 
 /**
 * Helper factory for creating media info objects.
+* @param {String} id - Media id
 * @param {MediaType} type - Media type
 * @param {SupportedMedia} [supported] - Supported media capabilities to be included on media info
 * @returns {MediaInfo}
 */
-MediaInfo.create = function(type,supported)
+MediaInfo.create = function(id,type,supported)
 {
+       //Support deprecated create(type,supported) 
+       if (!supported && Array.isArray(type))
+       {
+	       //Reallocate params
+	       // @ts-ignore
+	       supported = type;
+	        // @ts-ignore
+	       type = id;

```

Comment:
in general it's a little safer to use `@ts-expect-error` than `@ts-ignore`
```suggestion
	       // @ts-expect-error
	       supported = type;
	        // @ts-expect-error
	       type = id;
```